src/extension.js

```diff
@@ -13,47 +13,32 @@ function activate(context) {
 	const config = vscode.workspace.getConfiguration("synthwave84");
 
 	let disableGlow = config && config.disableGlow ? !!config.disableGlow : false;
-	
+	const workbencHtmlPath = config && config.workbencHtmlPath ? config.workbencHtmlPath : "";
+	if (workbencHtmlPath && !isValidPath(workbencHtmlPath)) return vscode.window.showInformationMessage(`The path provided is not valid: ${workbencHtmlPath}`);
+
 	let brightness = parseFloat(config.brightness) > 1 ? 1 : parseFloat(config.brightness);
 	brightness = brightness < 0 ? 0 : brightness;
 	brightness = isNaN(brightness) ? 0.45 : brightness;
 
 	const parsedBrightness = Math.floor(brightness * 255).toString(16).toUpperCase();
-	let neonBrightness = parsedBrightness;
-
-	let disposable = vscode.commands.registerCommand('synthwave84.enableNeon', function () {
-
-		const isWin = /^win/.test(process.platform);
-		const appDir = path.dirname(require.main.filename);
-		const base = appDir + (isWin ? "\\vs\\code" : "/vs/code");
-		const electronBase = isVSCodeBelowVersion("1.70.0") ? "electron-browser" : "electron-sandbox";
 
-		const htmlFile =
-			base +
-			(isWin
-				? "\\"+electronBase+"\\workbench\\workbench.html"
-				: "/"+electronBase+"/workbench/workbench.html");
+	const disposable = vscode.commands.registerCommand('synthwave84.enableNeon', function () {
 
-		const templateFile =
-				base +
-				(isWin
-					? "\\"+electronBase+"\\workbench\\neondreams.js"
-					: "/"+electronBase+"/workbench/neondreams.js");
+		const vscodeDir = path.dirname(require?.main?.filename || process.execPath);
+		const workbenchHtmlPath = workbencHtmlPath ? workbencHtmlPath : recursiveSearchPath(vscodeDir, "workbench", "html");
```

Comment:
it should rather be this
`workbencHtmlPath ??  recursiveSearchPath(vscodeDir, "workbench", "html");`
Otherwise `workbenchHtmlPath` might be null and then the split function in Line 29 will fail

Reply:
Hi @ireznik, you are right, I will address it at once, however I am going to take another path since the nullish coalescing operator only returns default values ​​if the value on the left is "null" or "undefined", if it is an empty text string, that is what it will return. 

This is the commit: cb3a64f6fb5af6e43cf887ad4eff959cdcfa3d42

Thank you for your cooperation.